packages/server/src/init.ts

```diff
@@ -267,6 +267,10 @@ async function init(config: PadlocConfig) {
         });
     }
 
+    if (config.directory.scim && !config.server.scimServerUrl) {
+        config.server.scimServerUrl = config.directory.scim.url;
+    }
```

Comment:
We need the url setting in `ScimServer` as well to construct the resource urls. This way we don't have to set it twice.